tests/query/parser/test_formula_mql_query.py

```diff
@@ -197,8 +197,8 @@ def test_simple_formula() -> None:
         ),
         keys=[
             JoinCondition(
-                left=JoinConditionExpression(table_alias="d0", column="time"),
-                right=JoinConditionExpression(table_alias="d1", column="time"),
+                left=JoinConditionExpression(table_alias="d1", column="time"),
```

Comment:
why did these need to be flipped?

Reply:
Because I changed the way these join conditions are generated [here](https://github.com/getsentry/snuba/pull/6068/files#diff-c8c2440ca4fc4b6d7f561461c07d868e3bef5a1c568f790e1b2f660f71dbd4d3R1229-R1243)

They are now in the same order as the JoinClause ([d1, then d0](https://github.com/getsentry/snuba/pull/6068/files#diff-3696392f4ee9019ab09a41c5fd761ccfbf842ef5ea713f30b6e40802032273baR189-R197)), which in my opinion makes more sense.